PrestaShop/PrestaShop
# [-]BO: Fix Online / Offline switch

## Description

Fix the offline / online button.
## Steps to Test this Fix

All browsers: Check if the label near the switch button on the product page correctly change when the product is online / offline.

Firefox: Check just if this button work^^
